openshift_cli_installer/utils/cli_utils.py

```diff
@@ -517,3 +518,16 @@ def prepare_clusters(clusters, ocm_token):
         _cluster["ocm-client"] = get_ocm_client(ocm_token=ocm_token, ocm_env=ocm_env)
 
     return clusters
+
+
+def click_echo(name, platform, section, msg, success=None, error=None):
+    if success:
+        fg = SUCCESS_LOG_COLOR
+    elif error:
+        fg = ERROR_LOG_COLOR
+    else:
+        fg = "white"
+
+    click.secho(
+        f"[Cluster: {name}, Platform: {platform},  Section: {section}]: {msg}", fg=fg
```

Comment:
how about 
`f"[Cluster: {name} - Platform: {platform} - Section: {section}]: {msg}"` for readability?